nixd/docs/editors/nvim-lsp.nix

```diff
@@ -11,7 +11,7 @@ let
       '';
 
       packages.myPlugins.start = with pkgs.vimPlugins; [
-        (nvim-treesitter.withPlugins (parsers: [ parsers.nix ]))
+        (nvim-treesitter.withPlugins (parsers: [ parsers.nix parsers.markdown parsers.markdown_inline ]))
```

Comment:
Format this line with [nixfmt-rfc-style](https://github.com/NixOS/nixfmt)

Reply:
Applied `$ nix fmt ./nixd/docs/editors/nvim-lsp.nix`